frontend/pages/sitemap/[language_code_dot_xml].js

```diff
@@ -111,14 +111,54 @@ const getStaticPageEntries = (staticPages) => {
 };
 
 const renderEntry = (BASE_URL, url, priority, changefreq, lastmod) => {
+  const languageCode = getLanguageCode(url);
   return `<url>
       <loc>${BASE_URL}${url}</loc>
+      ${getXHTMLLinkTag(BASE_URL, url, languageCode)}
       <priority>${priority ? priority : 1}</priority>
       <changefreq>${changefreq ? changefreq : "daily"}</changefreq>
       ${lastmod ? `<lastmod>${lastmod}</lastmod>` : ``}
     </url>`;
 };
 
+const getXHTMLLinkTag = (BASE_URL, url, languageCode) => {
+  if (languageCode === "en") {
+    return `<xhtml:link
+    rel="alternate"
+    hreflang="${languageCode}"
+    href="${BASE_URL}${url}"/>
+    <xhtml:link
+    rel="alternate"
+    hreflang="de"
+    href="${BASE_URL}/de${url}"/>`;
+  }
+  const englishURL = BASE_URL + "/" + getEnglishURLFromGerman(url);
+  if (languageCode === "de") {
+    return `<xhtml:link
+    rel="alternate"
+    hreflang="${languageCode}"
+    href="${BASE_URL}${url}"/>
+    <xhtml:link
+    rel="alternate"
+    hreflang="en"
+    href="${englishURL}"/>`;
+  }
+};
+
+const getLanguageCode = (url) => {
+  if (url.startsWith("/de/")) return "de";
+  if (url.startsWith("/en/")) return "en";
```

Comment:
I might be alone on this one (we should update our ESLint and Prettier rules to take a stance IMO), but I find that always including the braces is more readable and less bug prone. What do you think?
```suggestion
  if (url.startsWith("/de/")) {
    return "de";
  }
  
  if (url.startsWith("/en/")) { 
    return "en";
   }
```

Reply:
I've always preferred reading if statements with curly braces as you suggested :)